renderdoc/driver/vulkan/vk_pixelhistory.cpp

```diff
@@ -1067,7 +1071,12 @@ struct VulkanColorAndStencilCallback : public VulkanPixelHistoryCallback
     if(!m_Events.contains(eid) || !m_pDriver->IsCmdPrimary())
       return;
 
-    // TODO: can't end renderpass if we are not on the last subpass.
+    if(HasMultipleSubpasses())
+    {
+      RDCWARN("Multiple subpasses in a render pass are not supported for pixel history.");
```

Comment:
It would be good to set a bool member in the callback so we only print this message once, instead of for every draw.